components/common/SplashPageSearchBar/splashPageSearchBar.tsx

```diff
@@ -54,78 +60,76 @@ export const SplashPageSearchBar = (props: SearchProps) => {
-              startAdornment={
-                <InputAdornment position="start">
-                  <Search className="fill-primary text-4xl" />
-                </InputAdornment>
-              }
-            />
-          )}
-          renderOption={(props, option, { inputValue }) => {
-            const text = searchQueryLabel(option);
-            //add spaces between prefix and course number
-            const matches = match(
-              text,
-              inputValue
-                .replace(
-                  //CS1200 -> CS 1200
-                  /([a-zA-Z]{2,4})([0-9][0-9V]?[0-9]{0,2})/,
-                  '$1 $2',
-                )
-                .replace(
-                  //1200CS -> 1200 CS
-                  /([0-9][0-9V][0-9]{2})([a-zA-Z]{1,4})/,
-                  '$1 $2',
-                ),
-            );
-            const parts = parse(text, matches);
-            return (
-              <li {...props}>
-                {parts.map((part, index) => (
-                  <span
-                    key={index}
-                    className={
-                      'whitespace-pre-wrap' +
-                      (part.highlight ? ' font-bold' : '')
-                    }
-                  >
-                    {part.text}
-                  </span>
-                ))}
-              </li>
-            );
-          }}
+    <Autocomplete
+      autoHighlight={true}
+      className={props.className}
+      getOptionLabel={(option) => searchQueryLabel(option)}
+      options={options}
+      filterOptions={(options) => options}
+      // When a new option is selected return it to the parent
+      // component using selectSearchValue prop
+      onChange={(event: React.SyntheticEvent, newValue: SearchQuery | null) =>
+        props.selectSearchValue(newValue)
+      }
+      inputValue={inputValue}
+      onInputChange={(event, newInputValue) => {
+        setInputValue(newInputValue);
+      }}
+      renderInput={(params) => (
+        <InputBase
+          ref={params.InputProps.ref}
+          inputProps={params.inputProps}
+          className="rounded-md border-gray-300 dark:border-gray-700 border-2 w-full px-3 py-2 bg-white dark:bg-haiti placeholder-gray-700 dark:placeholder-gray-300 text-sm"
+          placeholder={
+            'ex.' +
+            ((props.searchBy ?? 'any') === 'any' ||
+            (props.searchBy ?? 'any') === 'course'
+              ? ' CS 1200'
+              : '') +
+            ((props.searchBy ?? 'any') === 'any' ||
+            (props.searchBy ?? 'any') === 'professor'
+              ? ' John Cole'
+              : '')
+          }
         />
-      </div>
-    </>
+      )}
+      renderOption={(props, option, { inputValue }) => {
+        const text = searchQueryLabel(option);
+        //add spaces between prefix and course number
+        const matches = match(
+          text,
+          inputValue
+            .replace(
+              //CS1200 -> CS 1200
```

Comment:
Is this working in practice?

Reply:
Yes, I believe this is just an indentation change here. This regex replace is in production